kernel_gateway/gatewayapp.py

```diff
@@ -198,12 +206,23 @@ def init_configurables(self):
         if self.seed_uri is not None:
             self.seed_notebook = self._load_notebook(self.seed_uri)
 
-        self.kernel_manager = SeedingMappingKernelManager(
-            parent=self,
-            log=self.log,
-            connection_dir=self.runtime_dir,
-            kernel_spec_manager=self.kernel_spec_manager
-        )
+        if self.api:
+            with open(self.api) as nb_fh:
+                notebook = nbformat.read(nb_fh, 4)
```

Comment:
Why is this not using `_load_notebook` which supports both local and remote notebooks and does the "do I have a valid kernel" check?